### PR TITLE
renders listing header and host w/ photo

### PR DESCRIPTION
**Pull request by elimoreh**

### CODE REVIEW

File: Module/client/src/styles/style.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import styled from 'styled-components';
```
Comment (sbtorres) [inline Module/client/src/styles/style.js]: Nice. Good refactor.

File: Module/client/src/components/app.jsx [hunk 2]
```diff
@@ -18,17 +21,22 @@ class App extends React.Component {
     axios.get('/listinginfo').then((response) => {
       let { id } = this.state;
       if (!id || (id > 100) || (id < 0)) { id = 0; }
-      this.setState({ data: response.data[id] });
+      this.setState({ data: response.data[id], loading: false });
```
Comment (sbtorres) [inline Module/client/src/components/app.jsx]: What does this do?

**Review by sbtorres — APPROVED** [review]

File: Module/client/src/components/app.jsx (hunk 2 above)
Comment (elimoreh) [reply Module/client/src/components/app.jsx]: It ensures the page will not render until the get request is completed and there will only be one get request... if theres a better way of doing this lmk. I originally assumed this was the job of componentDidMount()

Comment (sbtorres) [reply Module/client/src/components/app.jsx]: Unsure which is better honestly, might just be preference. I usually go with componentDidMount(). Up to you.

**Pull request merged**